pkg/controller/kfdef/const.go

```diff
@@ -13,27 +12,33 @@ const (
 var (
 	// watchedResources contains all resources we will watch and reconcile when changed
 	watchedResources = []schema.GroupVersionKind{
-		{Group: "apps", Version: "v1", Kind: "StatefulSet"},
-		{Group: "apps", Version: "v1", Kind: "Deployment"},
+		{Group: "rbac.authorization.k8s.io", Version: "v1", Kind: "ClusterRole"},
+		{Group: "rbac.authorization.k8s.io", Version: "v1", Kind: "ClusterRoleBinding"},
+		{Group: "", Version: "v1", Kind: "ConfigMap"},
+		{Group: "apiextensions.k8s.io", Version: "v1beta1", Kind: "CustomResourceDefinition"},
 		{Group: "apps", Version: "v1", Kind: "DaemonSet"},
-		{Group: "extensions", Version: "v1beta1", Kind: "Ingress"},
+		{Group: "apps", Version: "v1", Kind: "Deployment"},
 		{Group: "extensions", Version: "v1beta1", Kind: "Deployment"},
-		{Group: "", Version: "v1", Kind: "Service"},
-		{Group: "", Version: "v1", Kind: "ConfigMap"},
+		{Group: "extensions", Version: "v1beta1", Kind: "Ingress"},
+		{Group: "admissionregistration.k8s.io", Version: "v1beta1", Kind: "MutatingWebhookConfiguration"},
+		{Group: "", Version: "v1", Kind: "Namespace"},
 		{Group: "", Version: "v1", Kind: "PersistentVolumeClaim"},
+		{Group: "rbac.authorization.k8s.io", Version: "v1", Kind: "Role"},
+		{Group: "rbac.authorization.k8s.io", Version: "v1beta1", Kind: "Role"},
+		{Group: "rbac.authorization.k8s.io", Version: "v1", Kind: "RoleBinding"},
+		{Group: "rbac.authorization.k8s.io", Version: "v1beta1", Kind: "RoleBinding"},
 		{Group: "", Version: "v1", Kind: "Secret"},
+		{Group: "", Version: "v1", Kind: "Service"},
 		{Group: "", Version: "v1", Kind: "ServiceAccount"},
-		{Group: "rbac.authorization.k8s.io", Version: "v1beta1", Kind: "RoleBinding"},
-		{Group: "rbac.authorization.k8s.io", Version: "v1", Kind: "RoleBinding"},
-		{Group: "rbac.authorization.k8s.io", Version: "v1beta1", Kind: "Role"},
-		{Group: "rbac.authorization.k8s.io", Version: "v1", Kind: "Role"},
-		{Group: "admissionregistration.k8s.io", Version: "v1beta1", Kind: "MutatingWebhookConfiguration"},
+		{Group: "apps", Version: "v1", Kind: "StatefulSet"},
 		{Group: "admissionregistration.k8s.io", Version: "v1beta1", Kind: "ValidatingWebhookConfiguration"},
-		{Group: "rbac.authorization.k8s.io", Version: "v1", Kind: "ClusterRole"},
-		{Group: "rbac.authorization.k8s.io", Version: "v1", Kind: "ClusterRoleBinding"},
-		{Group: "apiextensions.k8s.io", Version: "v1beta1", Kind: "CustomResourceDefinition"},
 	}
 
-	// kfdefUIDMap maps the UID to KfDef Name
-	kfdefUIDMap = map[types.UID]types.NamespacedName{}
+	watchedKubeflowResources = []schema.GroupVersionKind{
+		{Group: "app.k8s.io", Version: "v1beta1", Kind: "Application"},
+		{Group: "rbac.istio.io", Version: "v1alpha1", Kind: "ServiceRole"},
+		{Group: "rbac.istio.io", Version: "v1alpha1", Kind: "ServiceRoleBinding"},
+		{Group: "networking.istio.io", Version: "v1alpha3", Kind: "VirtualService"},
+		{Group: "argoproj.io", Version: "v1alpha1", Kind: "Workflow"},
```

Comment:
Is there any default workflow deployed as part of Argo KFP? If we are doing it for Argo, we should also add Tekton alpha condition for Tekton as part of the watching list.
```go
{Group: "tekton.dev", Version: "v1alpha1", Kind: "Condition"},
```

Reply:
there is no default workflow.... I added the tekton.dev Condition.